integrations/Rockerbox/browser.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable prefer-destructuring */
+/* eslint-disable class-methods-use-this */
+import get from "get-value";
+import logger from "../../utils/logUtil";
+import { NAME } from "./constants";
+import { LOAD_ORIGIN } from "../ScriptLoader";
+import { getHashFromArray } from "../utils/commonUtils";
+
+class Rockerbox {
+  constructor(config) {
+    this.clientAuthId = config.clientAuthId;
+    this.name = NAME;
+    this.customDomain = config.customDomain;
+    this.enableCookieSync = config.enableCookieSync;
+    this.eventsMap = config.eventsMap || [];
+    this.useNativeSDKToSend = config.useNativeSDKToSend;
+  }
+
+  init() {
+    logger.debug("=== In init Rockerbox ===");
+    const host = this.customDomain ? this.customDomain : "getrockerbox.com";
+    const library =
+      this.customDomain && this.enableCookieSync ? "wxyz.rb" : "wxyz.v2";
+    (function (d, RB) {
+      if (!window.RB) {
+        window.RB = RB;
+        RB.queue = RB.queue || [];
+        RB.track =
+          RB.track ||
+          function () {
+            RB.queue.push(Array.prototype.slice.call(arguments));
+          };
+        RB.initialize = function (s) {
+          RB.source = s;
+        };
+        var a = d.createElement("script");
```

Comment:
How about using script loader utility?

Reply:
I wanted to customise parts of the scripts based on the config set in the UI, hence used directly.